guide/event-driven-and-pooling-consumer-service/event-driven-and-pooling-consumer.bal

```diff
@@ -0,0 +1,104 @@
+};
+documentation { Queue receiver endpoint for health check messages of the sensors. }
+endpoint mb:SimpleQueueReceiver queueReceiverHealth {
+    host: "localhost",
+    port: 5672,
+    queueName: "SensorHealth"
+};
+endpoint http:Client  HealthEP {
+    url: "http://localhost:8080/health"
+};
+documentation { Queue receiver endpoint for maintenance messages of sensors. }
+endpoint mb:SimpleQueueReceiver queueReceiverMaintenance {
+    host: "localhost",
+    port: 5672,
+    queueName: "Maintenance"
+};
+endpoint http:Client  MaintenanceEP {
+    url: "http://localhost:8080/maintenance"
+};
+documentation { Queue receiver endpoint for calibration messages. }
+endpoint mb:SimpleQueueReceiver queueReceiverCalibrate {
+    host: "localhost",
+    port: 5672,
+    queueName: "Calibration"
+};
+endpoint http:Client  CalibrateEP {
+    url: "http://localhost:8080/calibrate"
+};
+documentation { Service to receive messages to the new GeoActivities message queue. }
+service<mb:Consumer> geoListener bind queueReceiverGeoActivities {
+    documentation { Resource handler for new messages from queue. }
+    onMessage(endpoint consumer, mb:Message message) {
+        http:Request outRequest;
+        string messageText = check message.getTextMessageContent();
+        json requestPayload = messageText;
+        outRequest.setJsonPayload(untaint requestPayload);
+        var resp = GeoActivityEP->post("/", outRequest);
+        log:printInfo("Message recived from queue GeoActivities: " + messageText + "status:forwarded to /activity");
+    }
+}
+documentation { Service to receive messages to the SensorHealth message queue. }
+service<mb:Consumer> healthListener bind queueReceiverHealth {
+    documentation { Resource handler for new messages from queue. }
+    onMessage(endpoint consumer, mb:Message message) {
+        http:Request outRequest;
+        string messageText = check message.getTextMessageContent();
+        json requestPayload = messageText;
+        outRequest.setJsonPayload(untaint requestPayload);
+        var resp = HealthEP->post("/", outRequest);
+        log:printInfo("Message recived from queue SensorHealth: " + messageText + "status:forwarded to /health");
+    }
+}
+documentation { Service to receive messages to the Maintenance message queue. }
+service<mb:Consumer> maintenanceListener bind queueReceiverMaintenance {
+    documentation { Resource handler for new messages from queue. }
+    onMessage(endpoint consumer, mb:Message message) {
+        http:Request outRequest;
+        string messageText = check message.getTextMessageContent();
+        json requestPayload = messageText;
+        outRequest.setJsonPayload(untaint requestPayload);
+        var resp = MaintenanceEP->post("/", outRequest);
+        log:printInfo("Message recived from queue Maintenance: " + messageText + "status:forwarded to /maintenance");
+    }
+}
+documentation { Service to receive messages to the Calibration message queue. }
```

Comment:
No point in documenting services since API docs do not get generated for services

Reply:
So in ballerina how can I document a service ?

Reply:
I have searched on official documentation site but could not found any reference regarding the documenting services. Where can I find them?

Reply:
Already update to the new documentation syntax.